src/components/App/App.tsx

```diff
@@ -202,6 +204,9 @@ export default class App extends React.Component<AppProps, AppState> {
     this.loadSettings();
     this.loadYouTube();
     this.init();
+    if (this.props.isSubscriber) {
```

Comment:
we can just do this when firebase auth completes right? don't need to do it on mount/each update?